### PR TITLE
rename --repoid to --repo

### PR DESCRIPTION
**Pull request by MichaelMraka**

### CODE REVIEW

File: plugins/repoquery.py [hunk 1]
```diff
@@ -83,8 +84,11 @@ def parse_arguments(args):
     parser = dnfpluginscore.ArgumentParser(RepoQueryCommand.aliases[0])
     parser.add_argument('key', nargs='?',
                         help=_('the key to search for'))
-    parser.add_argument('--repoid', metavar='REPO',
+    parser.add_argument('--repo', metavar='REPO', action='append',
```
Comment (jsilhan) [inline plugins/repoquery.py]: not mention repoid in docs but support it, please.  i.e.: `parser.add_argument('--repo', '--repoid', metavar='REPO', action='append'`. we cannot drop it inmediatelly.

Comment (MichaelMraka) [inline plugins/repoquery.py]: Yes, otherwise it would be printed by --help/--help-cmd.

**Comment by timlau** [comment]
As it wrote in 
https://github.com/rpm-software-management/dnf-plugins-extras/pull/23

This would be an API break an inconsistance with all other tools there uses --repoid (both dnf/yum-utils)

**Comment by timlau** [comment]
dnf & plugins takes a lot of beating in the fedora community, and renaming well known option is a bad move at this point of time IMO

**Comment by MichaelMraka** [comment]
> This would be an API break an inconsistance with all other tools there uses --repoid (both dnf/yum-utils)

All dnf tools has been fixed to use consistent naming which is repo. Moreover --repoid does still work (for compatibility reasons) but is hidden from help.

**Comment by timlau** [comment]
as log as --repoid still work every thing shoud be ok

**Pull request merged**